lang/pt_BR.json

```diff
@@ -295,15 +295,15 @@
     "RetroAchievements.org does not condone or supply any copyright-protected ROMs to be used in conjunction with the supplied emulators.": "RetroAchievements.org não endossa nem fornece ROMs protegidas por direitos autorais para serem usadas em conjunto com os emuladores fornecidos.",
     "RetroAchievements.org's Terms of Use are subject to change.": "Os Termos de Uso do RetroAchievements.org estão sujeitos a alterações.",
     "RetroPoints: A measurement of rarity and estimated difficulty.": "RetroPoints: Uma medida de raridade e dificuldade estimada.",
-    "Revision": "Revision",
+    "Revision": "Revisão",
     "Search games": "Pesquisar jogos",
     "Search games...": "Pesquisar jogos...",
-    "See more": "See more",
-    "Select a file": "Select a file",
+    "See more": "Ver mais",
+    "Select a file": "Selecione um arquivo",
     "Select a game": "Selecione um jogo",
     "Select an achievement": "Selecione uma conquista",
-    "Select an emulator": "Select an emulator",
-    "Select an issue": "Select an issue",
+    "Select an emulator": "Selevione um Emulador",
```

Comment:
```suggestion
    "Select an emulator": "Selecione um Emulador",
```

Reply:
sorry, it was a writing error, but I already adjusted